src/components/page-layout/page-layout.js

```diff
@@ -8,7 +8,8 @@ class PageLayout extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      bottomBoundaryValue: 0
+      bottomBoundaryValue: 0,
+      enabled: false
```

Comment:
Might be worth giving this a more descriptive name for your future debugging self?

Reply:
Ohhh yes yes good idea.